packages/orbit-components/src/Table/TableBody/index.tsx

```diff
@@ -1,27 +1,11 @@
-"use client";
```

Comment:
Do we want to delete this? The same applies to all the sub-components.

Reply:
yep, it is not necessary. the component contains no hooks/state, or other constructs incompatible with RSCs